cloudferrylib/os/actions/check_cloud.py

```diff
@@ -12,153 +12,141 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+
 import contextlib
+import copy
 import time
 
+from cinderclient import exceptions as cinder_exc
+from glanceclient import exc as glance_exc
+from keystoneclient.openstack.common.apiclient import exceptions as ks_exc
+from neutronclient.common import exceptions as neutron_exc
+from novaclient import exceptions as nova_exc
+
 from cloudferrylib.base.action import action
-from cloudferrylib.utils import log
-from cloudferrylib.utils import proxy_client
-from cloudferrylib.os.identity.keystone import KeystoneIdentity
+from cloudferrylib.base import exception
 from cloudferrylib.os.compute.nova_compute import NovaCompute
-from cloudferrylib.os.network.neutron import NeutronNetwork
+from cloudferrylib.os.identity.keystone import KeystoneIdentity
 from cloudferrylib.os.image.glance_image import GlanceImage
+from cloudferrylib.os.network.neutron import NeutronNetwork
```

Comment:
I guess that would be better to import the modules here, not classes. What do you think about that?